packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -90,9 +90,12 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
     uint64 internal constant MAX_GAS_LIMIT = 200_000_000;
 
     /// @notice Fixed L2 gas overhead. Used as part of the L2 fee calculation.
+    ///         Deprecated since the Ecotone network upgrade
     uint256 public overhead;
```

Comment:
Since this is deprecated, will be renamed and replaced with a spacer at some point?

Reply:
I didn't want to delete it and turn it into a spacer in case we need to use it as part of a weird upgrade mechanism as the value is still passed into an event and picked up by `op-node` but its a noop

Reply:
See discussion here: https://github.com/ethereum-optimism/optimism/pull/10226/files#r1575444717

Basically if we ever upgrade L2 without upgrading L1, we now have extra data that we can pass into L2. This is helpful if for some reason there is a constraint that we cannot upgrade the L1 contracts in a timely manner

Reply:
Got it, thanks